stdlib/contrib/qxip/clickhouse/clickhouse.flux

```diff
@@ -0,0 +1,64 @@
+// Package clickhouse provides functions to query [ClickHouse](https://clickhouse.com/) using the ClickHouse HTTP API.
+//
+// ## Metadata
+// introduced: NEXT
+//
+package clickhouse
+
+import "csv"
+import "experimental"
+import "experimental/http/requests"
+
+// query queries data from ClickHouse using specified parameters.
+//
+// ## Parameters
+// - url: ClickHouse HTTP API URL. Default is `http://127.0.0.1:8123`.
+// - query: ClickHouse query to execute.
+// - limit: Query rows limit. Defaults is `100`.
+// - cors: Request remote CORS headers. Defaults is `1`.
+// - max_bytes: Query bytes limit. Default is `10000000`.
+// - format: Query format. Default is `CSVWithNames`.
+//
+//   _For information about available formats, see [ClickHouse formats](https://clickhouse.com/docs/en/interfaces/formats/)._
+//
+// ## Examples
+//
+// ### Query ClickHouse
+// ```no_run
+// import "contrib/qxip/clickhouse"
+//
+// option clickhouse.defaultURL = "https://play@play.clickhouse.com"
+//
+// clickhouse.query(query: "SELECT version()")
+// ```
+//
+// ## Metadata
+// tags: inputs
+//
+option defaultURL = "http://127.0.0.1:8123"
```

Comment:
This needs to be just after the import statement, and requires a comment above it describing the option:

```
// defaultURL is the default ClickHouse HTTP API URL.
option defaultURL = "http://127.0.0.1:8123"
```

Reply:
Thanks for the clarification. Fixed!